content/backend/typescript-apollo/1-getting-started.md

```diff
@@ -0,0 +1,464 @@
+
+<Instruction>
+
+Add the `generate` and `dev` script to your package.json
+
+```json{3-4}(path=".../hackernews-typescript/package.json")
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node-dev --transpile-only --no-notify --exit-child src/index.ts",
+    "generate": "ts-node --transpile-only src/schema.ts"
+  },
+```
+</Instruction>
+
+1. You can run `npm run generate` to update your `schema.graphql` and `nexus-typegen.ts` file when there are any changes in your `nexus` code.
+2. You can use `npm run dev` to start the web server and watch for any changes. 
+
+### Creating a GraphQL server
+
+Now that you have a schema, you can finally create your GraphQl server inside `index.ts`. 
+
+<Instruction>
+
+Write up the following code to start the server in `src/index.ts`. 
+
+```typescript(path=".../hackernews-typescript/src/index.ts")
+import { ApolloServer } from "apollo-server";
+
+// 1
+import { schema } from "./schema";
+export const server = new ApolloServer({
+    schema,
+});
+
+const port = 3000;
+// 2
+server.listen({port}).then(({ url }) => {
+    console.log(`🚀  Server ready at ${url}`);
+});
+
+```
+
+</Instruction>
+
+Let's understand what's going on here by walking through the numbered comments: 
+
+1. The schema object you created using `nexus` defines your GraphQL schema. You need to provide this when instantiating your server.
+
+2. You start the server and specify the port. After the server starts, it returns a `url` string inside a promise. 
+
+Now that your server is ready let's give it a run and see what happens!
+
+<Instruction> 
+
+Open up your terminal and run the following command
+
+```bash(path=".../hackernews-typescript/")
+npm run dev
+```
+
+</Instruction>
+
+If everything goes well, you should see the following output inside the terminal:
+
+
+```(nocopy)
+🚀  Server ready at http://localhost:3000/
+```
+
+Now if you navigate to `http://localhost:3000/` you should see the following page: 
+
+![Apollo Server Landing Page](https://i.imgur.com/2bOq8c1.png)
+
+If you click on the `Query your Server` button, you will be redirected to The Apollo Studio Explorer. This is an _online web-based GraphQL IDE_ for running queries and exploring your schema (along with many other advanced features). 
+
+> **Note:** If you'd prefer an _offline_ IDE that does not need access to the internet, there's a section covering this at the bottom of this chapter. 
+
+![Apollo Studio Explorer](https://i.imgur.com/jsDqCAi.png)
+
+In the schema tab placed on the left (the first icon under the apollo logo), you can see the entire GraphQL schema. There's not much there right now, but in the future it will come in handy to explore the details of your schema. 
```

Comment:
(In this case the visual anchor doesn't work because "Schema" is not spelled out on the tab, but I'd suggest to still stick to the convention)

```suggestion
In the **Schema** tab placed on the left (the first icon under the apollo logo), you can see the entire GraphQL schema. There's not much there right now, but in the future it will come in handy to explore the details of your schema. 
```

Reply:
```suggestion
In the schema tab placed on the left (the first icon under the Apollo logo), you can see the entire GraphQL schema. There's not much there right now, but in the future it will come in handy to explore the details of your schema. 
```